metaworld/envs/mujoco/sawyer_xyz/sawyer_assembly_peg.py

```diff
@@ -103,7 +103,7 @@ def reset_model(self):
         return self._get_obs()
 
     def _reset_hand(self):
-        for _ in range(10):
+        for _ in range(50):
```

Comment:
did something about the simulation change? why 50 vs 10?

Reply:
I really can't tell. When you watch the rendered simulation, it's clear that just 10 `_reset_hand` steps are no longer enough to consistently orient the hand as we want it (it ends up tilted when the policy starts). I searched through the XML to try to find out why this is the case, but I couldn't figure it out. Moving up to 50 steps solved the issue.

Reply:
Perhaps some masses or friction constants changed?

Does this mean you need to adjust the number of simulation steps elsewhere? e.g. for `step()`?

Reply:
Nope. It seems to only affect the grabber's initial orientation, so we only have to increase the steps in `_reset_hand`. I thought maybe the initial attributes got changed in the XML, but everything that references `hand` in the XML is identical to the old version. I have no idea what's going on

Reply:
I guess in theory it could be a friction/mass issue, but I would expect something like that to impact the scripted policy more (needing higher `max_path_length` or an increased `P` constant)